src/hooks/useOperatorOrder.ts

```diff
@@ -33,10 +75,18 @@ export function useOrder(orderId: string, updateInterval = 0): UseOrderResult {
       setError('')
 
       try {
-        const rawOrder = await getOrder({ networkId, orderId })
+        const { order: rawOrder, errorOrderPresentInNetworkId: errorOrderPresentInNetworkIdRaw } = await _getOrder(
+          networkId,
+          orderId,
+          NETWORK_ID_SEARCH_LIST.filter((network) => network != networkId),
```

Comment:
I think this filtering belongs in `_getOrder`, not on the caller.
And this parameter be optional, defaulting to all networks.

Reply:
I started doing like that for some reason. And I changed it to this, but I think u r right, I'll change it!